README.md

```diff
@@ -84,4 +82,4 @@ You can learn more in the [Create React App documentation](https://facebook.gith
 To learn React, check out the [React documentation](https://reactjs.org/).
 
 [prettier]: https://prettier.io/
-[TypeScript]: https://www.typescriptlang.org/
+[typescript]: https://www.typescriptlang.org/
```

Comment:
You've changed it to `typescript` here, but not changed the references in the doc.

Reply:
Looks like it works: https://github.com/mozilla/addons-code-manager/tree/minor-fixes-prettier

Reply:
It does work, but I thought we might want to be consistent with the casing anyway. 🤷‍♂️

Reply:
I think prettier enforces that for url keys because that's the spec. But TypeScript should be written with capital T and S.